template/doc.html

```diff
@@ -49,14 +51,6 @@
 
     function getDescription(schema) {
         let desc = schema.Description || '';
-        if (schema.Enum) {
-            desc += '\n' + `
-\`\`\`yaml
-# valid values
-${schema.Enum.map(e => `- '${e}'`).join('\n')}
-\`\`\``.trim()
```

Comment:
Does this mean Enum values will not be shown at all?

Reply:
I think this works for now :) If we get feedback that folks would like them called out explicitly then we can add back. Thanks again!